bin/GameIndex.dbf

```diff
@@ -4896,6 +4941,12 @@ Serial = SCUS-97212
 Name   = My Street (Online)
 Region = NTSC-U
 Compat = 4
+[patches = 23e24e93]
+	author=Kozarovv
+	// Adjusting M-bit timings a litle bit (still isn't perfect).
+	patch=1,EE,002ab7bc,word,01f9c97c
+	patch=1,EE,002ab7cc,word,200002ff
+[/patches]
```

Comment:
Not a good idea to include these yet, mbit pr isn't merged yet (which these patches require).